daq/runner.py

```diff
@@ -1009,6 +1016,11 @@ def _calculate_device_result(self, test_results):
                 if result not in ('pass', 'skip'):
                     failed = True
 
+        if len(device.host.enabled_tests) != len(processed):
```

Comment:
I think these 2 will never be equal since enabled_tests doesn't contain the startup stages like "acquire", "base" which are still included in module_results. A simple set include check should suffice.

Reply:
But it does work... ?  It's not listing all tests that are included in module_results, only the ones that have a result_code, so maybe that's why it works?